core/src/main/java/org/sonarsource/sonarlint/core/telemetry/TelemetryClient.java

```diff
@@ -32,17 +35,21 @@
 import java.nio.file.Path;
 
 public class TelemetryClient {
-  private static final String TELEMETRY_ENDPOINT = "https://chestnutsl.sonarsource.com";
   private static final String TELEMETRY_PATH = "telemetry";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "TELEMETRY_ENDPOINT" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068)